src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -1272,12 +1272,16 @@ enum VectorMask {
 
   // Vector Single-Width Floating-Point/Integer Type-Convert Instructions
   INSN(vfcvt_f_x_v,      0b1010111, 0b001, 0b00011, 0b010010);
+  INSN(vfcvt_f_xu_v,     0b1010111, 0b001, 0b00010, 0b010010);
   INSN(vfcvt_rtz_x_f_v,  0b1010111, 0b001, 0b00111, 0b010010);
+  INSN(vfcvt_rtz_xu_f_v, 0b1010111, 0b001, 0b00110, 0b010010);
 
   // Vector Widening Floating-Point/Integer Type-Convert Instructions
   INSN(vfwcvt_f_x_v,      0b1010111, 0b001, 0b01011, 0b010010);
+  INSN(vfwcvt_f_xu_v,     0b1010111, 0b001, 0b01010, 0b010010);
   INSN(vfwcvt_f_f_v,      0b1010111, 0b001, 0b01100, 0b010010);
   INSN(vfwcvt_rtz_x_f_v,  0b1010111, 0b001, 0b01111, 0b010010);
+  INSN(vfwcvt_rtz_xu_f_v, 0b1010111, 0b001, 0b01110, 0b010010);
```

Comment:
I see no use of these newly added assembler functions. So test coverage would be an issue. Maybe add them in the future when they are really needed?

Reply:
Sure, will fix.